pkgs/applications/virtualization/docker/default.nix

```diff
@@ -1,78 +1,156 @@
-{ stdenv, lib, fetchFromGitHub, makeWrapper, pkgconfig, go-md2man
-, go, containerd, runc
-, sqlite, iproute, bridge-utils, devicemapper, systemd
+{ stdenv, lib, fetchFromGitHub, makeWrapper, cmake, pkgconfig
+, go-md2man, libapparmor, apparmor-parser, libseccomp, git
```

Comment:
runc needs git (for the commit hash I guess). I really wish there was another way like what we do for docker/docker but it doesn't seem so…